src/services/openletters/index.jsx

```
@@ -471,6 +473,470 @@ export const AddOpenLetter = ({
+                  canvasProps={{
+                    width: 400,
+                    height: 200,
+                    className: "sigCanvas",
+                  }}
+                  ref={sigCanvas}
+                />
+
+                <button
+                  className="absolute top-3 right-0 bg-red-500 text-white px-2 py-1 w-30 rounded"
+                  onClick={handleClear}
+                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)